pkgs/development/tools/schemacrawler-bin/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+, jre
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
options: if you are interested
https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes

---

Thanks, added!